src/components/TeamCard/TeamCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import "./TeamCard.scss"
+import LiLogo from "./fancylinkedin.svg"
+import cat from "./cat.jpg"
+
+interface CardProps {
+  name: string
+  imgsrc: string
+  title: string
+}
+
+const TeamCard: React.FC<CardProps> = ({ name, imgsrc, title }) => (
+  <div className='Card'>
+    <img src={imgsrc} alt={name} className='Card__Image' />
+    <p className='Card__Name'>{name}</p>
+    <p className='Card__Title'>{title}</p>
```

Comment:
In general, let's avoid using `<p />` tags here. I believe you are using flex, so things might change when swapping to a div.